app/javascript/gobierto_dashboards/webapp/lib/config.js

```diff
@@ -1,20 +1,20 @@
 export const contractsColumns = [
-  {field: 'assignee', translation: I18n.t('gobierto_dashboards.dashboards.contracts.assignee'), format: null},
-  {field: 'title', translation: I18n.t('gobierto_dashboards.dashboards.contracts.contractor'), format: 'truncated'},
-  {field: 'final_amount_no_taxes', translation: I18n.t('gobierto_dashboards.dashboards.contracts.final_amount_no_taxes'), format: 'currency'},
-  {field: 'start_date', translation: I18n.t('gobierto_dashboards.dashboards.contracts.date'), format: null},
+  {field: 'assignee', translation: I18n.t('gobierto_dashboards.dashboards.contracts.assignee'), format: null, cssClass: ''},
```

Comment:
This file is becoming a jumble. What is this file for?

Reply:
Yeah, I know :( we decided earlier that it will be separated in two files (one config for each dashboard) as soon as there isn't as many pending branches to merge.